onnxruntime/python/session.py

```diff
@@ -50,6 +67,17 @@ def get_modelmeta(self):
         "Return the metadata. See :class:`onnxruntime.ModelMetadata`."
         return self._model_meta
 
+    def get_providers(self):
+        "Return list of registered execution providers."
+        return self._providers
+
+    def set_providers(self, providers):
```

Comment:
possibly. i'm open to changing it. but i thought get/set made more sense here. (because of the get api which exists because of the default registration)